databricks.yml

```
@@ -17,3 +18,134 @@ targets:
+            python_file: "notebooks/week5/01.preprocess.py"
+            parameters:
+              - "--root_path"
+              - ${var.root_path}
+          libraries:
+           - whl: ./dist/*.whl
+        - task_key: if_refreshed
+          condition_task:
+            op: "EQUAL_TO"
+            left: "{{tasks.preprocessing.values.refreshed}}"
+            right: "1"
+          depends_on: 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)